code/modules/atmospherics/machinery/components/gas_recipe_machines/atmos_machines_recipes.dm

```diff
@@ -54,23 +54,14 @@ GLOBAL_LIST_INIT(gas_recipe_meta, gas_recipes_list())
 	requirements = list(/datum/gas/hydrogen = 300, /datum/gas/bz = 50)
 	products = list(/obj/item/stack/sheet/mineral/metal_hydrogen = 1)
 
-/datum/gas_recipe/crystallizer/healium_grenade
-	id = "healium_g"
-	name = "Healium crystal"
-	min_temp = 200
-	max_temp = 400
-	energy_release = -2000000
-	requirements = list(/datum/gas/healium = 100, /datum/gas/oxygen = 120, /datum/gas/plasma = 50)
-	products = list(/obj/item/grenade/gas_crystal/healium_crystal = 1)
-
-/datum/gas_recipe/crystallizer/proto_nitrate_grenade
-	id = "proto_nitrate_g"
-	name = "Proto nitrate crystal"
-	min_temp = 200
-	max_temp = 400
-	energy_release = 1500000
-	requirements = list(/datum/gas/proto_nitrate = 100, /datum/gas/nitrogen = 80, /datum/gas/oxygen = 80)
-	products = list(/obj/item/grenade/gas_crystal/proto_nitrate_crystal = 1)
+/datum/gas_recipe/crystallizer/cleansing_crystal
+	id = "cleanse_g"
+	name = "Cleansing crystal"
+	min_temp = 100
+	max_temp = 300
+	energy_release = 3000000
+	requirements = list(/datum/gas/nitrogen = 500, /datum/gas/oxygen = 800, /datum/gas/plasma = 1500, /datum/gas/nitrous_oxide = 700)
```

Comment:
we may want to change these things in future once crystalizer stuff is fleshed out further. still not totally sure about it but I understand not wanting to prevent any creation ever